README.md

```diff
@@ -141,3 +141,48 @@ This is the identifier of the sound you have told to play and you can use it to
 ```
 gdbus call --session --dest com.endlessm.HackSoundServer --object-path /com/endlessm/HackSoundServer --method com.endlessm.HackSoundServer.StopSound a72276d2-a856-4531-aac1-59fe1d331fc1
 ```
+
+# Logging
+
+## Log levels
+The default log level is WARNING. The log level can be set using the environment variable `LOGLEVEL`.
+
+To log everything:
+```
+LOGLEVEL=0 flatpak run com.endlessm.HackSoundServer
+```
+
+To log levels from INFO and upper
+```
+LOGLEVEL=INFO flatpak run com.endlessm.HackSoundServer
+```
+
+Log levels from the Python logging system are used. These are the following:
```

Comment:
I think we don't need to include this table. In README documentation we should try to cover the common cases that people will want to look up quickly. Otherwise the risk is too much information to read through, then people won't read it :stuck_out_tongue_closed_eyes: